packages/wdio-types/src/Capabilities.ts

```diff
@@ -154,6 +154,8 @@ export interface VendorExtensions extends EdgeCapabilities, AppiumW3CCapabilitie
     'sauce:visual'?: SauceLabsVisualCapabilities
     // Experitest Access Keys
     'experitest:accessKey'?: string
+    //LambdaTest w3c specific
+    'LT:Options'?: LambdaTestCapabilities
```

Comment:
Is lowercase/uppercase important here? I would recommend to keep caps lowercased so wanted to make sure this isn't a platform requirement

Reply:
This must be in uppercase as this is a LambdaTest platform requirement.

Reply:
Ouch, ok